Development.md

```diff
@@ -42,19 +42,15 @@ Why is this necessary? The project requires a customized "copy resources" build
 ## Running Tests
 **NOTE**: Several of the tests require a mongod instance to be running on the default host/port, `localhost:27017`.
 
-Additionally, please note that each benchmark test runs for a minimum of 1 minute and therefore **the entire benchmark suite will take around 20-30 minutes to complete**.
-
 You can run tests from Xcode as usual. If you prefer to test from the command line, keep reading.
 
 ### From the Command Line 
-Tests can be run from the command line with `make test`. By default, this will run all the tests excluding the benchmarks.
+Tests can be run from the command line with `make test`. By default, this will run all the tests.
```

Comment:
I may not be understanding the consequences of this PR, but can't we just remove this now and tell users to run `swift test`, since that will no longer run the benchmarks? This has the added benefit of making things more uniform and tied with just Swift as opposed to involving Make targets.

Reply:
Got it!